pages/dkp/kaptain/2.1.0/user-management/active-directory/index.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: layout.pug
+navigationTitle: Active Directory
+title: Authenticate users using Active Directory
+menuWeight: 30
+excerpt: Authenticate users to access Kaptain with Active Directory (AD) groups.
+beta: false
+enterprise: false
+---
+
+Authenticate users to access Kaptain with Microsoft’s Active Directory (AD) groups.
+
+Kaptain relies on Dex as its identity provider, a service that enables the management of users and user groups. Dex, in turn, delegates to one or more external identity providers like AD.
+
+The Active Directory (AD) feature allows you to manage and control access to your Kaptain instance, by using an existing rule set of permissions established in the AD (via the AD admin interface) to authenticate and grant access to Kaptain users and user groups.
+
+You can activate AD to authenticate users regardless of your environment set up (air-gapped or networked), or infrastructure provider.
+
+If you are already using AD groups to authenticate users in DKP or other platforms, this feature will allow you to have centralized access management and control. It will also allow your users to access Kaptains's Kubeflow dashboard with the same credentials they have been using for other platforms.
+
+When the AD is active, users will be able to log in to Kaptain with their AD credentials. For this, Kaptain’s dex uses LDAP to call up and verify the entered credentials against your existing AD groups.
+
+As with all matters regarding security, we encourage you to thoroughly review permissions set by the AD in conjunction with your security team.
+
+## Prerequisites
+
+- You have configured an AD server.
+- The automatic creation of profiles in your Kaptain instance is [enabled][automatic] or disabled. If disabled, you must manually [create the profiles][manual] you wish to authenticate before activating the AD.
+- You have administrative access to a configured DKP cluster via `kubectl`.
+- You have administrative access to the DKP UI.
+
+## Set up AD in Kaptain
+
+1.  Set up an authentication protocol with Dex for [LDAP][LDAP]. When doing so, adapt the connector fields as necessary.
```

Comment:
This seems really vague.   Also, I don't see how creating a connector will enable anything to do with Kaptain, you still have to do something to connect Kaptain to DEX, right?   (which is the stuff documented in the 'external-dex' page.)

In general, rather than going into how to configure LDAP here, I think it would be better to link this page the Kommander docs that describe how to configure DEX (https://docs.d2iq.com/dkp/kommander/2.2/security/oidc/) or better, to add a new page to the Kommander docs that explain how to configure dex to talk to LDAP (like we had in 1.8 with this page: https://docs.d2iq.com/dkp/konvoy/1.8/access-authentication/howto-dex-ldap-connector/) and then link to *that* page (as well as the one that explains how to config the dex client)

This page should only discuss the Kaptain specific portions of any LDAP customizations (like the customization of what groups will have access to Kaptain).